orchagent/crmorch.cpp

```diff
@@ -839,3 +1115,10 @@ string CrmOrch::getCrmAclTableKey(sai_object_id_t id)
     ss << "ACL_TABLE_STATS:" << "0x" << std::hex << id;
     return ss.str();
 }
+
+string CrmOrch::getCrmDashAclGroupKey(sai_object_id_t id)
```

Comment:
What is this function for? Can you please have a comment and an example for easy reference?

Reply:
Added comment